wordpress-mobile/WordPress-iOS
# Stats Insights: Caching improvements + dismiss Ghost cards when InsightStore State from CoreData is nil

This PR adds functionality to ensure that Ghost cards will be dismissed when retrieving insight data from CoreData is nil

Ref #17904

Testing:
1. Choose the Fieldguide site
2. Go to Stats screen and then click Insights tab.
3. Screen should load normally
4. Navigate / back out to home screen
5. Go to stats screen and click Insights tab again
6. No ghost cards should appear

## Regression Notes
1. Potential unintended areas of impact
StatsInsights

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested and ran UITests

3. What automated tests I added (or what prevented me from doing so) 
None

PR submission checklist:
